snapcraft/internal/build_providers/_base_provider.py

```diff
@@ -172,7 +172,9 @@ def destroy(self) -> None:
         """
 
     @abc.abstractmethod
-    def _run(self, command: Sequence[str]) -> Optional[bytes]:
+    def _run(
+        self, command: Sequence[str], hide_output: bool = False
```

Comment:
strange that implementing classes did not barf on this

Reply:
Yes I thought so as well...